pkg/diagnostics/consts/consts.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package consts
+
+import (
+	semconv "go.opentelemetry.io/otel/semconv/v1.10.0"
+)
+
+const (
+	// DaprInternalSpanAttrPrefix is the internal span attribution prefix.
+	// Middleware will not populate it if the span key starts with this prefix.
+	DaprInternalSpanAttrPrefix = "__dapr."
+	// DaprAPISpanNameInternal is the internal attribution, but not populated to span attribution.
+	DaprAPISpanNameInternal = DaprInternalSpanAttrPrefix + "spanname"
+
+	// Span attribute keys
+	// Reference trace semantics https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/trace/semantic_conventions
+	DBSystemSpanAttributeKey                 = string(semconv.DBSystemKey)
+	DBNameSpanAttributeKey                   = string(semconv.DBNameKey)
+	DBStatementSpanAttributeKey              = string(semconv.DBStatementKey)
+	DBConnectionStringSpanAttributeKey       = string(semconv.DBConnectionStringKey)
+	MessagingSystemSpanAttributeKey          = string(semconv.MessagingSystemKey)
+	MessagingDestinationSpanAttributeKey     = string(semconv.MessagingDestinationKey)
+	MessagingDestinationKindSpanAttributeKey = string(semconv.MessagingDestinationKindKey)
+	GrpcServiceSpanAttributeKey              = string(semconv.RPCServiceKey)
+	NetPeerNameSpanAttributeKey              = string(semconv.NetPeerNameKey)
+
+	DaprAPISpanAttributeKey           = "dapr.api"
```

Comment:
Question: Why are these values not in the form of `io.dapr.x` or `x.dapr.io`

Reply:
I don’t know - this didn’t change